tests/Jellyfin.Model.Tests/Dlna/StreamBuilderTests.cs

```diff
@@ -403,32 +403,39 @@ public async Task BuildVideoItemWithDirectPlayExplicitStreams(string deviceName,
                             stream => Assert.DoesNotContain(stream.Codec, val.VideoCodecs));
                     }
 
-                    // todo: fill out tests here
+                    // TODO: Fill out tests here
                 }
 
                 // DirectStream and Remux
                 else
                 {
-                    // check expected video codec (1)
+                    // Check expected video codec (1)
                     Assert.Contains(targetVideoStream.Codec, val.TargetVideoCodec);
                     Assert.Single(val.TargetVideoCodec);
 
                     if (transcodeMode == "DirectStream")
                     {
+                        // Check expected audio codecs (1)
                         if (!targetAudioStream.IsExternal)
                         {
-                            // check expected audio codecs (1)
-                            Assert.DoesNotContain(targetAudioStream.Codec, val.AudioCodecs);
+                            if (val.TranscodeReasons == TranscodeReason.ContainerNotSupported)
```

Comment:
Shouldn't it be like this?
```suggestion
                            if ((val.TranscodeReasons & TranscodeReason.ContainerNotSupported) != 0)
```